src/components/ProjectsGrid.tsx

```diff
@@ -32,53 +34,58 @@ import stairs from "../assets/project-page/project-icons/stairs.png";
 import statue from "../assets/project-page/project-icons/statue.png";
 import styled from "styled-components";
 
-const ProjectLinkWrapper = styled.div<LayoutProps & GridProps & FlexboxProps>`
+const ProjectLinkWrapper = styled.div<
+  LayoutProps & GridProps & FlexboxProps & SpaceProps
+>`
   ${layout};
   ${grid};
   ${flexbox};
+  ${space};
   display: flex;
   align-items: center;
+  transition: 1s;
```

Comment:
this is redundant

Reply:
Nope, this goes back to the snappy and static thing, the transition just gives that playful animation feel on the grid when moving between different screen sizes

Reply:
I know not everyone flits between screen sizes but just to cover all bases we may as well